regression/tests/whitelabel/test_user_account.py

```diff
@@ -77,7 +70,7 @@ def test_reset_password(self):  # TE-2044
         self.login_page.visit()
         self.login_page.send_forgot_password(self.user_email)
         self.assertTrue(
-            self.login_page.is_password_reset_email_message_visible()
+            self.login_page.is_password_reset_email_message_visible
```

Comment:
is this by mistake ?

Reply:
is_password_reset_email_message_visible is property.